odoo/addons/test_assetsbundle/tests/test_assetsbundle.py

```diff
@@ -632,3 +660,340 @@ def test_01_debug_mode_assets(self):
+            var d=4;
+            '''
+        )
+
+    def test_04_globmanifest_with_irasset(self):
+        view = self.make_asset_view('test_assetsbundle.manifest4')
+        self.env['ir.asset'].create({
+            'name': 'test_jsfile4',
+            'bundle': 'test_assetsbundle.manifest4',
+            'glob': 'test_assetsbundle/static/src/js/test_jsfile1.js',
+        })
+        view._render()
+        attach = self.env['ir.attachment'].search([('name', 'ilike', 'test_assetsbundle.manifest4.js')], order='create_date DESC', limit=1)
+        content = attach.raw.decode()
+        self.assertStringEqual(
+            content,
+            '''
+            /* /test_assetsbundle/static/src/js/test_jsfile3.js defined in bundle 'test_assetsbundle.manifest4' */
+            var c=3;;
+
+            /* /test_assetsbundle/static/src/js/test_jsfile1.js defined in bundle 'test_assetsbundle.manifest4' */
+            var a=1;
+            '''
+        )
+
+    def test_05_only_irasset(self):
+        view = self.make_asset_view('test_assetsbundle.irasset1')
+        self.env['ir.asset'].create({
+            'name': 'test_jsfile4',
+            'bundle': 'test_assetsbundle.irasset1',
+            'glob': 'test_assetsbundle/static/src/js/test_jsfile1.js',
+        })
+        view._render()
+        attach = self.env['ir.attachment'].search([('name', 'ilike', 'test_assetsbundle.irasset1.js')], order='create_date DESC', limit=1)
+        content = attach.raw.decode()
+        self.assertStringEqual(
+            content,
+            '''
+            /* /test_assetsbundle/static/src/js/test_jsfile1.js defined in bundle 'test_assetsbundle.irasset1' */
+            var a=1;
+            '''
+        )
+
+    def test_06_replace(self):
+        view = self.make_asset_view('test_assetsbundle.manifest1')
+        self.env['ir.asset'].create({
+            'name': 'test_jsfile4',
+            'bundle': 'test_assetsbundle.manifest1',
+            'directive': 'replace',
+            'target': 'test_assetsbundle/static/src/js/test_jsfile1.js',
+            'glob': 'http://test.external.link/javascript1.js',
+        })
+        view._render()
+        attach = self.env['ir.attachment'].search([('name', 'ilike', 'test_assetsbundle.manifest1.js')], order='create_date DESC', limit=1)
+        content = attach.raw.decode()
+        self.assertStringEqual(
+            content,
+            '''
+            /* /test_assetsbundle/static/src/js/test_jsfile2.js defined in bundle 'test_assetsbundle.manifest1' */
+            var b=2;;
+
+            /* /test_assetsbundle/static/src/js/test_jsfile3.js defined in bundle 'test_assetsbundle.manifest1' */
+            var c=3;;
+
+            /* /test_assetsbundle/static/src/js/test_jsfile4.js defined in bundle 'test_assetsbundle.manifest1' */
+            var d=4;
+            '''
+        )
+        # LPE FIXME: where is my http glob ?
```

Comment:
@Arcasias 
Hey, There are a few blindspots in those tests, including this line.
Do you mind checking it to refine the specs ?